packages/calcite-components/src/components/tip-manager/tip-manager.scss

```diff
@@ -65,15 +71,14 @@
     p-4;
   animation-name: none;
   animation-duration: var(--calcite-animation-timing);
-  block-size: var(--calcite-tip-manager-height);
+  block-size: var(--calcite-internal-tip-manager-block-size, 19vh);
```

Comment:
`--calcite-internal-tip-manager-block-size` doesn't seem to be defined internally, so it can probably be removed.